plugin/nodes/image.py

```diff
@@ -10,18 +10,14 @@ def build_radl_image(config):
 
     id = get_child(dictionary=config, key='id', required=True)
     storage = get_child(dictionary=config, key='storage')
-    username = get_child(dictionary=config, key='username', required=True)
-    password = get_child(dictionary=config, key='password', required=True)
-    public_key = get_child(dictionary=config, key='public_key', required=True)
-    private_key = get_child(dictionary=config, key='private_key', required=True)
+    username = get_child(dictionary=config, key='username') or 'user'
```

Comment:
As I said, this should always be defined in the dictionary.

Reply:
It is very unlikely that, if you left the username empty, the user in the image will be 'user'. So leaving it blank would lead to an error.

The idea is that, when you define an image, you need to know the default user that the image defines.

Therefore it should be mandatory (and reflect that on the README), unless you say that the plugin is creating the user when deploying the virtual machine. If it is the second case, that behaviour should be explained in the README as well.

Reply:
Yes, the second assumption is the right case. The user is created during contextualization